### PR TITLE
Update copyright on each page to 2019

### PR DESCRIPTION
**Pull request by aweisberg**
Closes: #30

### CODE REVIEW

File: website/static/community.html [hunk 1]
```diff
@@ -96,7 +96,7 @@ <h2>Starburst Newsletter</h2>
     </div>
 </div>
 
-<div class="footer">&copy; Copyright 2013-2016, Facebook</div>
+<div class="footer">&copy; Copyright 2013-2019, Facebook</div>
```
Comment (JoelMarcey) [inline website/static/community.html]: could maybe use "present" to avoid changing all the time?

**Comment by aweisberg** [comment]
IANAL but that isn't how copyright notices are supposed to work. They are supposed to be the year the work was created and if it spans multiple years (such as it's updated several times) it's supposed to be range since it was last edited.

In practice people do some weird things like update copyright notices to include the current year on things where nothing has changed. Docusaurus just puts in the current year. I think this problem will go away once all pages are rendered by Docusaurus so I would rather just stick with the current year.

If we haven't switch to Docusaurus by the end of 2019 I will be sad.

**Comment by JoelMarcey** [comment]
IANAL either. I thought that you could use `present`, but actually, with our license headers now, we do not put a year at all. Ping me internally and I can show you.

But, yeah, Docusaurus would handle the copyright part of this for you.

**Comment by caniszczyk** [comment]
This was added by Brian.

**Pull request closed without merging**